src/Scrabble/Move/Move.hs

```diff
@@ -42,9 +42,13 @@ createMove' :: Validator -- ^
 createMove' validate b (Rack rack) wp dict = if valid then go else errMsg where
   errMsg        = Left "error: rack missing input letters"
   rackLetters   = fmap letter rack
-  valid         = containsAll (toString putLetters) (toString rackLetters)
+  valid         = containsAllWithBlanks (toString putLetters) (toString rackLetters)
+  blanksUsed    = countBlanksUsed (toString putLetters) (toString rackLetters)
   putLetters    = letter <$> wordPutTiles wp
-  rackRemainder = fmap fromLetter $ foldl' (flip delete) rackLetters putLetters
+  rackRemainder = fmap fromLetter $
+                  applyNTimes blanksUsed (delete Blank) $
+                  foldl' (flip delete) rackLetters putLetters
+  applyNTimes n f = foldr (.) id (replicate n f)
```

Comment:
Are you ok with the behavior here? This is the part where, lets say if your rack is "_POPS", and you play "POP_", with "S" as your choice of letter for the blank, rack remainder becomes "_" instead of "S", so you get to reuse the blank tile.